rpg/Global/GlobalCommands.cs

```diff
@@ -39,6 +49,83 @@ public void GetStatistics(Client sender)
                     // TODO: Cop ranks, experience
                 }
             }
+        }
+
+        [Command("loc")]
+        public void ShowLocationCommand(Client sender)
+        {
+            var pos = API.getEntityPosition(sender.handle);
+
+            API.sendChatMessageToPlayer(sender, pos.X + ", " + pos.Y + ", " + pos.Z);
+        }
+
+        [Command("me")]
+        public void emotes(Client sender, string text)
+        {
+            string name = "~p~" + sender.name +" " + text;
+            API.sendChatMessageToAll(name);
+
+        }
+
+        [Command("pm")]
+        public void message(Client sender, Client reciever, string text)
+        {
+            API.sendChatMessageToPlayer(reciever, "~r~"+ sender.name + " tells you ", text);
+
+        }
+
+        [Command("shout")]
+        public void shout(Client sender, string text)
+        {
+
+            API.sendChatMessageToAll("~h~~b~" + sender.name + " shouts " + text);
+
+        }
+
+        [Command("give")]
+        public void givemoney (Client sender,Client player,int amount) 
+         {
+
+            if (API.getEntityData(sender, "Money") > amount && amount>0)
+            {
+
+                API.setEntityData(player, "Money", amount+API.getEntityData(player, "Money"));
+                API.sendChatMessageToPlayer(player, "~g~"+sender.name+" Sent you "+amount);
+                API.setEntityData(sender, "Money", API.getEntityData(sender, "Money")-amount);
+            }
+
+            else API.sendChatMessageToPlayer(sender, "~r~You don't have enough money");
+
+         }
+
+        [Command("help")]
+        public void help (Client sender)
+        {
+
+            API.sendChatMessageToPlayer(sender, "/me /pm /kill /shout /give /stop /surrender /time /stats");

```

Comment:
those are the commends for general use, havent included stuff like /payfine /arrest because they are documented else where, meaning people with the interest of using those commands, probably already know them from other means such as collision shapes,or helpers.